integration-tests/spring-boot-properties/src/main/java/io/quarkus/it/spring/boot/InterfacePropertiesResource.java

```diff
@@ -0,0 +1,18 @@
+package io.quarkus.it.spring.boot;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+
+@Path("/interface")
+public class InterfacePropertiesResource {
+
+    @Inject
+    private InterfaceProperties properties;
```

Comment:
Can we drop `private` from here (and the other instances in the integration test application)? The reason is that quarkus logs a warning for injections into `private` fields and we could use the cleanest test output we can get.

Reply:
Didn't know that. Why is package-private preferred to private?

Reply:
Because Quarkus then doesn't need to use reflection to set the field. You can read more details here: https://quarkus.io/guides/cdi-reference#native-executables-and-private-members